analyzer/src/main/kotlin/PackageManager.kt

```diff
@@ -255,7 +255,7 @@ abstract class PackageManager(
                     val errors = listOf(
                         createAndLogIssue(
                             source = managerName,
-                            message = "Resolving dependencies for '$definitionFile' failed with: " +
+                            message = "Resolving $managerName dependencies for '$definitionFile' failed with: " +
```

Comment:
Commit message:

A word seems to be missing in "matching  with resolutions unambiguously". Or at least there's a double-space in there.

Also, isn't the package manager clear from the definition file? I.e. can you given an example where the same definition file is use by different package managers? Are you e.g. referring to `package.json` eventually being used by both NPM and Yarn?

Reply:
> A word seems to be missing in "matching with resolutions unambiguously". Or at least there's a double-space in there.

Done.

> Also, isn't the package manager clear from the definition file? I.e. can you given an example where the same definition file is use by different package managers? Are you e.g. referring to package.json eventually being used by both NPM and Yarn?

`package.json` would be an example, yes. As package managers are plugin-able, it could be any definition file.

Reply:
> As package managers are plugin-able, it could be any definition file.

That argument does not hold as it's ORT looking for the definition files here, not the package manager.

Reply:
> That argument does not hold as it's ORT looking for the definition files here, not the package manager.

Do you prefer to ditch the commit, or do you want a different commit message?

Reply:
I'm undecided. I currently do not really see a value in this change, esp. as at least the issue source already contains the package manager name. If you'd like to keep the commit, please at least improve the commit messaging by naming the NPM vs Yarn example. Then it'd be fine with me.

Reply:
One more thought: If dropping the last commit would get rid of the "breaking change" label (as issue resolutions do not need to be adjusted), I'd have a *slight* tendency towards dropping it.

Reply:
Ok great,I still have a preference towards keeping it. I've updated the commit message accordingly.

Reply:
> One more thought: If dropping the last commit would get rid of the "breaking change" label (as issue resolutions do not need to be adjusted), I'd have a slight tendency towards dropping it.

Already the first commit modifies the issue message, so the breaking change would remain even if dropping that commit IIUC.